src/modules/classes.js

```diff
@@ -0,0 +1,230 @@
+// all methods
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-loop-func */
+export default class classes {
+    static popupWindow = document.querySelector('#popup-window')
+
+  static baseurl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Nvwt41EYsSFQ5YyQ8wBC'
+
+    static getInvolvementApi = async () => {
+      const apiUrl = `${this.baseurl}CpnE9NNUbu1zv9OE8RDw/likes`;
+      const getUrl = fetch(apiUrl, {
+        method: 'GET',
+      });
+      const getItems = await getUrl;
+      const test = await getItems.json();
+      return test;
+    }
+
+      static popupId = async (e) => {
+        let id = '';
+        const arrayItem = await this.getFoodItems();
+        arrayItem.forEach((element) => {
+          if (element.idMeal === e) {
+            id = element.idMeal;
+          }
```

Comment:
While you did an impressive job working on the project requirements I noticed you disabled eslint in this file. It is against project requirements and microverse policy to disable eslint or any other linters. So it would be nice if you fixed this issue by enabling eslint.